pretrained_models/README.md

```diff
@@ -70,11 +77,15 @@ Model Predictions
 
 </table>
 
-## Generating predictions with DeepCell.org
+## Running pretrained models in the cloud
 
-Deepcell.org is a web-based interface to access our pre-trained deep learning models. The website allows you to easily upload example images, run them on our available models, and download the results without requiring any local installation.
+Deep learning models perform best when run on GPU hardware. However, having every lab purchase their own GPU is not reasonable or efficient. To help make it as easy as possible to run our deep learning models, we have created a number of different options for using our cloud-based servers to analyze data. This makes it easy for anyone to submit their images to be analyzed, without needing to worry about installing any complicated software or purchase any expensive hardware.
```

Comment:
"accelerated hardware (like GPUs)" instead of "GPU hardware."
Might also change it so that it reads, "not every lab has the resources or desire to purchase and manage their own GPU-enabled workstations"